index.html

```diff
@@ -4,18 +4,218 @@
+                    <p>-</p>
+                    <button class="btn">Shop Collection</button>
+                </div>
+                
+                
+                    <img src="../images/plant-img1.webp" alt="">
+                
+                
+            </div>
+            <div class="image-card">
+                <div class="image-texts">
+                    <h3>PLANTS</h3>
+                    <p>-</p>
+                    <button class="btn">Shop Collection</button>
+                </div>
+                
+                <img src="../images/plant-img14.webp" alt="plant-image">
+            </div>
+            <div class="image-card">
+                <div class="image-texts">
+                    <h3>SUCULLENTS</h3>
+                    <p>-</p>
+                    <button class="btn">Shop Collection</button>
+                </div>
+                
+                <img src="../images/plant-img3.webp" alt="">
+            </div>
+            
+        </section>
+        <section class="about-us"> 
+            <div class="abt">
+                <h4>ABOUT US</h4>
+                <p>-</p>
+                <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a great place for you to tell a story and let your users know a little more about you.
+
+
+                </p><br><br>
+                <p>This is a great space to write long text about your company and your services. You can use this space to go into a little more detail about your company. Talk about your team and what services you provide.</p>
+            </div>
+            
+            <div class="img1">
+                <img src="../images/corridor-img.webp" alt="corridor image">
+            </div>
+            <div class="img2">
+                <img src="../images/leaf-img.webp" alt="leaf-image">
+            </div>
+            <div class="img3">
+                <img src="../images/garden-img.webp" alt="garden image">
+            </div>
+            
+            <div class="buy-online">
+                <div>
+                    <p>BUY ONLINE NOW & GET 10% OFF</p>
+                    <button>Shop Now</button>
+                </div>
+                
+            </div>
+        </section>
+        
+        <section class="form">
+            <form action="">
+                <p class="form-heading tc">FOR SPECIAL REQUESTS & ORDERS</p>
+                <p class="tc">-</p>
+            
+                <label for="name">First Name *</label><br>
+                    <input type="text" id="firstname">
+                
+                <label for="lastname">Last Name</label><br>
+                    <input type="text" id="lastname">
+                
+                <label for="email">Email *</label><br>
+                    <input type="text" id="email">
```

Comment:
You have added stars to indicate that the fields are mandatory but there is no actual validation being done on form submit, how might we do this?